vignettes/package.Rmd

```diff
@@ -0,0 +1,345 @@
+---
+title: "Using Rust code in R packages"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Using Rust code in R packages}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+The rextendr package provides two utility functions for developing R packages with
```

Comment:
@yutannihilation,
may I suggest replacing "the rextendr package" with "rextendr" or `rextendr` or any other phrase that does not contain "package"?
There are several examples of such sentences throughout the text, and if we can avoid repetition, it will sound better.

Reply:
Oh, for the phrase "the extendr package," I just followed the other vignette. I have no sense which is better.

https://github.com/extendr/rextendr/blob/00ce4650ee044e4b7c66ebeaecaa65c6d5ec7024/vignettes/rmarkdown.Rmd#L15

@clauswilke 
What do you think?

Reply:
@yutannihilation, I understand that. My point was when you write "the rextendr *package* ... for developing R *packages*", you repeat "package". The closer two "package" usages are, the stranger it sounds. It is perfectly ok to use "The rextendr package", I am just not 100% sure it is the best choice when it is immediately followed by another "package"/"packages" word.


One more example is here

> The package doesn't depend on the rextendr package

It is always challenging to avoid such repetition, but if you can, it is much easier to read, at least for some people, which is probably related to the cultural background/native languages of the reader.

Reply:
Ah, thanks. So, maybe it's rather the latter "pacakges" that I need to fix? Or rewrite the whole sentence?

> if you can

I think I can, to some extent, but the problem is it takes me more time to polish the language and I don't have that bandwidth at the moment unfortunately. That's one of the reasons why I originally published this as an "unofficial" one on my personal blog. I'll try to find time to do this, but, if I fail, let me just merge this. You can improve or completely rewrite this after that :)

Reply:
Anyway, I really appreciate for pointing out these problems in my writings. Feel free to suggest more drastic rewrites! (though I'm not sure if I can address these in this pull request)

Reply:
@yutannihilation, sure, I understand, I am no native speaker myself.
If @clauswilke approves, I have no objections. Feel free to modify this PR or make another to improve the wording.